Gemfile.lock

```diff
@@ -565,9 +561,8 @@ GEM
     roadie-rails (3.0.0)
       railties (>= 5.1, < 7.1)
       roadie (~> 5.0)
-    rodf (1.1.1)
+    rodf (1.2.0)
```

Comment:
Not sure to understand why this dependencies is upgraded. 
@mkllnk any thoughts?

Reply:
Because rodf is a dependency of spreadsheet_architect.